### ISSUE TITLE
doesnt work on kali?

### ISSUE THREAD

**Issue by NFS11** [EntySec/CamOver#27]
after using the `pip3 install git+https://github.com/EntySec/CamOver` it for some reason doesnt work when i type in camover in terminal. however when i do it in termux it works but doesnt find any vunerable ips. does anyone know why? i try camover and then i see this afterwards: `camover: command not found`. any help would be appreciated. thank you

**Comment by enty8080** [comment]
Are you sure that path containing pip3 installed files is in your `$PATH`?

**Comment by NFS11** [comment]
@enty8080 I have, directly into my home folder

**Comment by Xia0-0** [comment]
因为kali的安全策略，现在不允许直接用 pip 安装到系统 Python 环境，所以用上面的指令
sudo apt install pipx
pipx ensurepath
pipx install git+https://github.com/EntySec/CamOver

然后稍等几分钟，等待安装，视网络环境而定。

使用：
camover -a 目标IP

camover -t --shodan  *************************  
自己的会员API密钥，需购买付费

有可能会误报，自己挨个试下

**Comment by NFS11** [comment]
@Xia0-0 I solved it, thanks!

**Comment by alexhale23456-dev** [comment]
hello , is this helpful to get the credentials of cameras or not

**Comment by NFS11** [comment]
@alexhale23456-dev yeah it works I tried it on a vulnerable camera I bought for testing. however note that it only works on vulnerable cameras and it doesn't bring you the live feed, just the credentials (which I assume is the only thing you want) for you to input where needed.
